src/utils/format/index.ts

```diff
@@ -58,3 +58,6 @@ export const toQueryString = (value: {
   Object.entries(value)
     .reduce((query, [key, value]) => `${query}${key}=${value}&`, '?')
     .slice(0, -1)
+
+export const localeCurrencyToNumber = (value: string) =>
+  Number(value.replace(',', ''))
```

Comment:
```suggestion
  Number(value.replaceAll(',', ''))
```
혹은
```suggestion
  Number(value.replace(/,/g, ''))
```
로 되어야 올바르게 동작할 것 같습니다!

Reply:
[변경완료](https://github.com/price-offer/offer-fe/pull/179/commits/17abbec909f742c99514d4e2c6f59e8da13e77e8) !